AlphaWallet/Settings/ViewModels/ServerViewModel.swift

```diff
@@ -13,6 +13,21 @@ protocol ServerTableViewCellViewModelType {
     var selectionStyle: UITableViewCell.SelectionStyle { get set }
 }
 
+protocol ServerImageTableViewCellViewModelType {
```

Comment:
Is a protocol necessary?

Reply:
I'm just following the style of the previous code.

Reply:
OK. I don't know if you have changed it, but in the future, let's not overdo it and set a new (better) example then :)